server/repository/src/db_diesel/preference.rs

```diff
@@ -73,25 +74,23 @@ impl<'a> PreferenceRepository<'a> {
         //    "{}",
         //    diesel::debug_query::<DBType, _>(&final_query).to_string()
         //);
-        let result = final_query
-            .load::<PreferenceJoin>(self.connection.lock().connection())?
-            .into_iter()
-            .map(|(preference_row, _store_row)| Preference { preference_row })
-            .collect();
+        let result = final_query.load::<PreferenceRow>(self.connection.lock().connection())?;
 
         Ok(result)
     }
 }
 
-type BoxedPreferenceQuery = IntoBoxed<'static, InnerJoin<preference::table, store::table>, DBType>;
```

Comment:
The inner join needed to be left join (inner will filter out any results where no store ID set) but store fields weren't used, and I didn't think they would be, so cleaned up while I was here